Tiltfile

```diff
@@ -37,7 +48,7 @@ local_resource(
 )
 
 docker_build_with_restart(
-    'localhost:5000/egressd',
+    'egressd',
```

Comment:
hm, it doesn't work for me when i didn't set `TILT_DEFAULT_REGISTRY`, let's maybe use this approach then?
```
registry = os.environ.get("TILT_DEFAULT_REGISTRY", 'localhost:5000')
default_registry(registry)
```